### ISSUE TITLE
Security of the game

### ISSUE THREAD

**Issue by CelineBoudier** [ocadotechnology/aimmo#173]
Very important due to the nature of our game (user code running in the backend) and the fact that we're dealing with users which will be children.

**Comment by OlafSzmidt** [comment]
Perhaps related to #114 and #115.

**Comment by OlafSzmidt** [comment]
Relevant

**Comment by CelineBoudier** [comment]
not stale

**Comment by j4mesholland** [comment]
not stale

**Comment by faucomte97** [comment]
Should we add #833, #834, #835 and #839 to this epic?

**Comment by faucomte97** [comment]
All current issues for this epic are in QA. However we might want to have another meeting to discuss what might still be relevant to this epic?

**Comment by CelineBoudier** [comment]
yes good idea! I'll create one or we can just move the technical chapter and talk :)
